miio/device.py

```diff
@@ -57,6 +58,7 @@ def __init__(
         self.token: Optional[str] = token
         self._model: Optional[str] = model
         self._info: Optional[DeviceInfo] = None
+        self._actions: Optional[List[ActionDescriptor]] = None
```

Comment:
This should be `Dict[str, ActionDescriptor]` to be consistent with to-be-added `_settings` and `_sensors` which should be taken care of in a separate PR. Then we have the descriptors nicely cached after the initial status/info call! :-)

Let me know if you want to prepare a PR for that, otherwise I will do it when I find some free cycles (I'm currently busy with the genericmiot stuff).

Reply:
I will make a sperate PR for caching, I am now realizing how messy and big the original PR was :)

Reply:
Is it correct that the key schould be the `method_name`?

Reply:
I checked it and I do think the key schould indeed be the method_name